release_build.sh

```diff
@@ -25,6 +24,16 @@ fi
 KBS_VERS_ARRAY=(${KBS_VERS})
 printf "%s\n" "${KBS_VERS_ARRAY[@]}"
 
+echo "Fetching list of latest CRI-O patch releases"
+echo "================================================="
+git clone https://github.com/cri-o/cri-o
```

Comment:
You could get a sorted list of tags without cloning using `git ls-remote --tags --sort=-v:refname https://github.com/cri-o/cri-o`. You probably could store the output in the file and then process it further to get the latest point release for each kubernetes major version.

But your solution works too, so just saying.